src/03-modality-agnostic-files.md

```diff
@@ -333,9 +334,10 @@ questionnaire).
 Template:
 
 ```Text
-sub-<label>/[ses-<label>/]
-    sub-<label>[_ses-<label>]_scans.tsv
-    sub-<label>[_ses-<label>]_scans.json
+sub-<label>/
+    [ses-<label>/]
+        sub-<label>[_ses-<label>]_scans.tsv
+        sub-<label>[_ses-<label>]_scans.json
```

Comment:
I actually preferred this the old way, and thought that was the general style. Are we conforming to an otherwise dominant convention here?

While it's not an issue in this case, for longer templates the extra indentation pushes more of the template off-screen and requires scrolling.

Reply:
It's easy enough to change in the rendering code, although I _think_ we were planning to switch to a tree-like style, if we can figure out how to do it programmatically (see #807).

Reply:
FWIW unless we're trying to show things at the subject level and session level relative to one another, I would prefer that we combine the two directories on one line for the sake of saving horizontal space.

Reply:
I was just adopting the way other templates are rendered by the macros. 

Happy to leave it on a single line if this is how templates will be rendered.

The idea about the "tree-like" style was more for data example than the templates: to also make examples and templates "look" different.

Reply:
If this is how macros are rendering, that's fine. I'd like to change that, but they can all be changed together.

Reply:
Let's make a mental not to adapt this on manually when we update the templates